docs/src/starknet/account.md

```diff
@@ -0,0 +1,49 @@
+# Creating And Deploying Accounts
+
+Starknet Foundry cast supports entire account management flow with the `sncast account create` and `sncast account deploy` commands.
+
+Do the following to start interacting with the Starknet:
+
+- create account with the `sncast account create` command
+
+    ```shell
+    $ sncast \
+      --url http://127.0.0.1:5050
+      --network testnet \
+      account create \
+      --name some-name
+      
+    Account successfully created. Prefund generated address with at least 432300000000 tokens. It is good to send more in the case of higher demand, max_fee * 2 = 864600000000
+    command: Create account
+    max_fee: 0x64a7168300
+    address: 0x7a949e83b243068d0cbedd8d5b8b32fafea66c54de23c40e68b126b5c845b61
+    ```
+
+    You can also pass common `--accounts-file` argument with a path to (existing or not existing) file where you want to save account info.
+    
+    For a detailed CLI description, see [account create command reference](../appendix/cast/account/create.md).
+
+
+- prefund generated address with tokens
+  
+    You can do it both by sending tokens from another starknet account or by bridging them with StarkGate.
```

Comment:
No-one decided, so let's - I think this might not be a bad idea to link to StarkGate. :+1: